pkg/virt-launcher/virtwrap/util/libvirt_helper.go

```diff
@@ -374,7 +375,7 @@ func SetupLibvirt() error {
 	if err != nil {
 		return err
 	}
-	defer qemuConf.Close()
+	defer fileClose(qemuConf, &err)
```

Comment:
You count here on `err` to be defined for you, but it might change and then we will hit a nil pointer here. Also, why wouldn't you just return the error from `fileClose`?

Reply:
Yes indeed I count on err be defined, but this is no different from any other place in all KubeVirt code where you take an address of variable someone defined earlier.  If someone changes the code he would need to take care of all implications.  If you think it would be better I can explicitly define and set it in the beginning .

The whole purpose of fileClose is to be called on `defer` you can return any value from it (or use that value). 
 I use that function to assign value to the  `err` variable which is then returned